core/block_dragger.js

```diff
@@ -106,18 +115,23 @@ Blockly.BlockDragger = function(block, workspace) {
 
 /**
  * Start dragging a block.  This includes moving it to the drag surface.
+ * @param {!goog.math.Coordinate} currentDragDeltaXY How far the pointer has
+ *     moved from the position at mouse down, in pixel coordinates.
  */
-Blockly.BlockDragger.prototype.startBlockDrag = function() {
+Blockly.BlockDragger.prototype.startBlockDrag = function(currentDragDeltaXY) {
```

Comment:
But it's only called at the start of a drag, which is clearly specified by the name of the function.